evo1/atb/predict.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from evo1.atb.entity import ATBEntityStats
+from memory.rng import EvolandRNG
+
+logger = logging.getLogger(__name__)
+
+
+class AttackPrediction:
```

Comment:
This can predict the damage of an attack, in or out of combat (out of combat it's used to predict the first hit).